Task3/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Task 3</title>
+
+        <link rel="stylesheet" type="text/css" href="css/app.css">
+
+        <script src="js/constants.js"></script>
+        <script src="js/zombies.js"></script>
+    </head>
+    <body>
+        <div class="zombies">
+        </div>
+        <div class="zombies-status">
+            <span>0</span> of <span>0</span>
+        </div>
+
+        <script src="js/components/Zombie.js" defer></script>
```

Comment:
you dont need a "defer" attribute here
it would make sense if you put all this scripts (Zombie.js, ZombieSmall.js) in <head>